ern-api-impl-gen/src/generators/ApiImplGen.js

```diff
@@ -102,6 +123,13 @@ export default class ApiImplGen {
       }
     })
   }
+
+  updatePackageJsonWithApiNames (outputDirectoryPath: string, apis: Array<Object>) {
+    const packageJsonPath = path.join(outputDirectoryPath, 'package.json')
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'))
+    packageJson.ern.containerGen.apiNames = _.map(apis, (api) => { return api.apiName })
```

Comment:
`_map` function call can be shortened to 
```javascript
 _.map(apis, api => api.apiName)
```
- If only one param the parenthesis are not needed
- If only one statement, curly braces should not be used and the arrow function will return the evaluated statement